src/petals/utils/packaging.py

```diff
@@ -0,0 +1,50 @@
+import re
+from typing import Any
+
+import torch
+from hivemind import nested_flatten, nested_pack
+
+# TODO: Move functions to hivemind
+
+
+def _mark_masked_tensor(index: int):
```

Comment:
```suggestion
def _mark_masked_tensor(index: int) -> bytes:
```

Please complete the type annotations at least in the cases where the type is known for sure.